classes/02class/exercises/c02-network06/afaryy/README.md

```diff
@@ -0,0 +1,91 @@
+# c02-network06
+
+
+- Create an Elastic IP:
+```bash
+export AWS_PROFILE=devopsacademyau
+aws sts get-caller-identity
+
+EIP_ID=$(aws ec2 allocate-address --domain vpc \
+    --query AllocationId --output text)
+aws ec2 describe-addresses  --allocation-ids $EIP_ID
+# Output:
+{
+    "Addresses": [
+        {
+            "PublicIp": "52.64.188.172",
+            "AllocationId": "eipalloc-0b8f804b7f7c2684d",
+            "Domain": "vpc",
+            "PublicIpv4Pool": "amazon",
+            "NetworkBorderGroup": "ap-southeast-2"
+        }
+    ]
+}
+
+
+```
+- Creating a NAT gateway and attach it to a subnet with an Elastic IP
+```bash
+PUBLIC_SUBNET_NAME='da-c02-public-a';
+SUBNET_ID=$(aws ec2 describe-subnets  \
+    --filter "Name=tag:Name,Values=$PUBLIC_SUBNET_NAME" \
+    --query 'Subnets[].SubnetId' --output text);
+ECHO $SUBNET_ID
```

Comment:
In linux, `echo` must be lowercase. This probably works on Windows but ideally we try to use the linux commands.

The variable names like `SUBNET_ID` are fine in uppercase, only the `ECHO` should be `echo`.

Reply:
Recently, I prefer git bash. I try to use different tools. Both ECHO and echo are ok in git bash. I don't think it is a big problem to finish the excercises by trying different tools in diffrerent OS. However, I will consider to use cygwin and avoid using ECHO later. Thanks.